cmd/k8s-operator/operator.go

```diff
@@ -77,6 +77,7 @@ func main() {
 		tsNamespace           = defaultEnv("OPERATOR_NAMESPACE", "")
 		tslogging             = defaultEnv("OPERATOR_LOGGING", "info")
 		image                 = defaultEnv("PROXY_IMAGE", "tailscale/tailscale:latest")
+		k8sProxyImage         = defaultEnv("K8S_PROXY_IMAGE", "tailscale/k8s-proxy:latest")
```

Comment:
Question: are we ready to immediately publish this to stable track? I'm not opinionated either way, as you think is better, but if we only published to unstable for N releases, then those images are typically tagged with `unstable`.

Reply:
I'm currently planning to publish to stable. Happy to change course if anyone raises concerns though.